### ISSUE TITLE
'Attach files' has no indicator that I've attached a file

### ISSUE THREAD

**Issue by christianbundy** [fraction/oasis#523]
## What's the problem you want solved?

When you click 'attach files' and try to upload a file, there's no way to know that you've actually attached a file. Also, since it says "files" I'd assume you can do multiple, but when I hit 'attach files' twice and hit preview then only the latest file is there.

## Is there a solution you'd like to recommend?

At the very least maybe change it to "attach file" (singular) and add some documentation about how it should be used?

**Comment by samebchase** [comment]
Yup, I ran into this today. Does Oasis support attachment upload? I remember reading a message by an Oasis author saying that they use Patchwork to upload images because Oasis cannot.

**Comment by christianbundy** [comment]
Yup, just hit 'attach files' and then hit 'preview. 👍
